src/knx/platform.h

```diff
@@ -49,21 +54,60 @@ class Platform
     virtual void setupSpi();
     virtual void closeSpi();
     virtual int readWriteSpi(uint8_t *data, size_t len);
-#if 0
-    // Flash memory
-    virtual size_t flashEraseBlockSize(); // in pages
-    virtual size_t flashPageSize();       // in bytes
-    virtual uint8_t* userFlashStart();   // start of user flash aligned to start of an erase block
-    virtual size_t userFlashSizeEraseBlocks(); // in eraseBlocks
-    virtual void flashErase(uint16_t eraseBlockNum); //relativ to userFlashStart
-    virtual void flashWritePage(uint16_t pageNumber, uint8_t* data); //write a single page to flash (pageNumber relative to userFashStart
-#endif
-    virtual uint8_t* getEepromBuffer(uint16_t size) = 0;
-    virtual void commitToEeprom() = 0;
+
+    //Memory
+
+    // --- Legacy support only. Do not use for new plattforms ---
+    // --- can be remove if all plattforms have been changed to support the NonVolatileMemory functions
+    virtual uint8_t* getEepromBuffer(uint16_t size);
+    virtual void commitToEeprom();
+      // -------------------------------------------------------------------------------------------------------
+
+    virtual uint8_t* getNonVolatileMemoryStart();
```

Comment:
Maybe we could use __attribute__ ((deprecated)) or something like that?

Reply:
as i wrote that comment I did not realize that the EEPROM Emulation API is also usefull for implementing a plattform with a RAM-Buffered Flash.
I added the option to use ram buffered flash in the rp2040 plattform by using the EEPROM-API.
So I would now remove change the comment to respect that

Reply:
Ok.

Reply:
done